### ISSUE TITLE
Sidenav component cannot run in my project

### ISSUE THREAD

**Issue by beidoufe** [JackGit/material-ui-vue#3]
This sidenav UI completely fits my demands, but some problem came to me when I try to apply it in my project.
Here is my deps:
"devDependencies": {
        "babel-core": "^6.13.2",
        "babel-loader": "^6.2.4",
        "babel-plugin-transform-runtime": "^6.12.0",
        "babel-preset-es2015": "^6.13.2",
        "babel-runtime": "^5.8.38",
        "css-loader": "^0.23.1",
        "less": "^2.7.1",
        "less-loader": "^2.2.3",
        "file-loader": "^0.8.1",
        "style-loader": "^0.8.3",
        "url-loader": "^0.5.5",
        "vue-hot-reload-api": "^2.0.6",
        "vue-html-loader": "^1.2.3",
        "vue-loader": "^8.5.3",
        "vue-style-loader": "^1.0.0",
        "vue-resource": "^0.9.3",
        "webpack": "^1.13.1",
        "webpack-dev-server": "^1.15.0",
        "moment": "^2.9.0",
        "jquery": "^2.1.1",
        "daterangepicker": "^0.0.3",
        "bootstrap": "^3.3.7",
        "font-awesome": "^4.3.0",
        "underscore": "^1.8.3",
        "vue-spinner": "^1.0.2",
        "material-ui-vue": "^0.1.4",
        "materialize-css": "^0.97.7"
    }

Basically I have every dep installed in my project.
But, when I import modules like this:

import SideNav from 'material-ui-vue/components/side-nav/side-nav.vue';
import NavCollapsibleItem from 'material-ui-vue/components/navbar/nav-collapsible-item.vue';
import NavDropdownItem from 'material-ui-vue/components/navbar/nav-dropdown-item.vue';
import NavItem from 'material-ui-vue/components/navbar/nav-item.vue';
import CollapsibleHeader from 'material-ui-vue/components/collapsible/collapsible-header.vue';
import CollapsibleBody from 'material-ui-vue/components/collapsible/collapsible-body.vue';

Then I run webpack:

![image](https://cloud.githubusercontent.com/assets/7550810/17999082/4518cffc-6baa-11e6-8a66-15517aa89d5d.png)

**Comment by beidoufe** [comment]
Oh if I only import this two modules:
import SideNav from 'material-ui-vue/components/side-nav/side-nav.vue';
import NavItem from 'material-ui-vue/components/navbar/nav-item.vue';
That is completely OK, nothing went haywire.
